e2e_test/sink/kafka/create_sink.slt

```diff
@@ -148,13 +153,13 @@ create sink invalid_pk_column from t_kafka with (
 
 statement ok
 insert into t_kafka values
-    (1, '8DfUFencLe', 31031, 1872, 1872, 26261.416, 23956.39329760601, '2023-04-14 06:27:14.104742', '\x00'),
-    (2, 'sIo1XXVeHZ', 22690, 4598, 4598, 27031.225, 31923.077305746086, '2023-04-13 23:44:04.869742', '\x00'),
-    (3, 'LVLAhd1pQv', 5985, 5894, 14894, 2660.2903, 9742.475509566086, '2023-04-13 23:36:51.269742', '\x000000'),
-    (4, 'ORjwy3oMNb', 6306, 7406, 24962, 21217.777, 3119.719721891862, '2023-04-14 01:12:07.993742', '\xDEADBEEF'),
-    (5, 'sSkKswxrYd', 22765, 9253, 9253, 22749.5, 17464.91553421121, '2023-04-14 03:57:22.324742', '\xFEEDBEEF'),
-    (6, 'V4y71v4Gip', 4014, 10844, 28842, 5885.368, 11210.458724794062, '2023-04-13 10:42:02.137742', '\xCAFEBABE'),
-    (7, 'YIVLnWxHyf', 10324, 12652, 15914, 3946.7434, 10967.182297153104, '2023-04-14 04:41:03.083742', '\xBABEC0DE'),
-    (8, 'lv7Eq3g8hx', 194, 19036, 28641, 13652.073, 993.408963466774, '2023-04-13 13:52:09.356742', '\xDEADBABE'),
-    (9, 'nwRq4zejSQ', 10028, 20090, 24837, 20699.559, 11615.276406159757, '2023-04-13 12:40:42.487742', '\xDEADBABE'),
-    (10, '0oVqRIHqkb', 26951, 20674, 20674, 19387.238, 9042.404483827515, '2023-04-13 16:40:58.888742', '\x00');
+    (1, '8DfUFencLe', 31031, 1872, 1872, 26261.416, 23956.39329760601, '2023-04-14 06:27:14.104742', '\x00', '0 second', '0001-01-01', '00:00:01.123456', '0001-01-01 00:00:00.123456'::timestamptz, '{}'),
```

Comment:
`'0001-01-01 00:00:00.123456'::timestamptz` is discouraged usage. It is better to include a timezone:
* `'0001-01-01 00:00:00.123456Z'`
* `'0001-01-01 00:00:00.123456+00:00'`
* `'0001-01-01 00:00:00.123456+08:00'`

Reply:
```sql
create table test (
  tz timestamptz
);

insert into test values ('0001-01-01 00:00:00Z');
```
 will cause ` Parse error: Can't cast string to timestamp (expected format is YYYY-MM-DD HH:MM:SS[.D+{up to 6 digits}] or YYYY-MM-DD HH:MM or YYYY-MM-DD or ISO 8601 format)`.
 `'1969-01-01 00:00:00Z'` is ok.